src/dropdown/list/scroll-custom-event.interface.ts

```diff
@@ -0,0 +1,27 @@
+import { Scroll } from "@angular/router";
```

Comment:
I don't think we want to use this event here 🤔 I think we can just use the generic `Event` interface (based on this that seems to be correct: https://developer.mozilla.org/en-US/docs/Web/API/Document/scroll_event there's no speicifc `ScrollEvent`)

Reply:
Ok, changing :D